.github/workflows/build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           repository: opensearch-project/OpenSearch
-          ref: feature/extensions
+          ref: main
```

Comment:
We dont need to publish to maven local as OpenSearch snapshots[1] will be published couple of times a day. 

[1] https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/

Reply:
Just removed it